Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift

```diff
@@ -25,7 +25,7 @@ final class ContinueButton: UIButton {
         let view = UILabel()
         view.text = viewModel.initials
         view.isAccessibilityElement = false
-        view.font = UIFont.boldSystemFont(ofSize: 17)
+        view.font = UIFont(name: Inter.bold.rawValue, size: 17)
```

Comment:
Suggestion: Maybe make an extension of the `UIFont` initializer that takes a case of the Inter enum? Or make a use of a protocol that Inter and other fonts can conform to and use that in an initializer? That way we don't have to write `.rawValue` everywhere

Reply:
The "Inter" font is only being used by the Norwegian news apps today. Why are we using it in a SDK component shared by many different Schibsted apps?

It'll look out of place in Aftonbladet, SvD, Bilbasen, etc.

Imo. a component like the simplified login should just stick to using San Fransisco.

Reply:
Ok, I have an explanation. The Inter font is the standard Schibsted font across the brands. It aligns with the strategy to make the Schibsted account(company) recognizable by the brand's users. The same font is used for account, privacy, and payment gateway services. This is the valid Schibsted font.
The new implementation of cookie consent banner from Bilbasen also uses Inter font. You can see it [here ](https://www.figma.com/file/1AxYNn0zEck0aEGcaHkqYi/Simplified-login-%E2%80%A2-Native-iOS-SDK%2FDesktop?node-id=1092%3A22995)

@clausjoergensen @oscarsilver

Reply:
Alright, that's a acceptable reason ;)